xarray/tests/test_backends.py

```diff
@@ -3524,11 +3567,26 @@ def create_zarr_target(self):
             yield tmp
 
     @contextlib.contextmanager
-    def create_store(self):
+    def create_store(self, **kwargs):
         with self.create_zarr_target() as store_target:
-            group = backends.ZarrStore.open_group(store_target, mode="a")
+            group = backends.ZarrStore.open_group(store_target, mode="a", **kwargs)
             yield group
 
+    def test_write_store(self) -> None:
```

Comment:
does this need to be over-ridden still if the default is to turn the cache _off_

Reply:
I think so, because we need to explicitly ensure that the cache is off, otherwise tests will fail. If we rely on the default behavior (which is defined very far away from this test), and if that default value changes, then tests will fail here and it will not be clear why.

Reply:
another way to remove this code would be to change the tests so that they no longer rely on the old `ZarrStore` behavior; maybe this could be a separate PR.

Reply:
I'll push a commit. The `create_store` on this class seems pointless.